finality-aleph/src/new_network/substrate.rs

```diff
@@ -21,19 +21,74 @@ impl<B: Block, H: ExHashT> RequestBlocks<B> for Arc<NetworkService<B, H>> {
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+pub enum SendError {
+    NotConnectedToPeer(PeerId),
+    LostConnectionToPeer(PeerId),
+    LostConnectionToPeerReady(PeerId),
+    SendTimeout(PeerId, u64),
+}
+
+const SEND_TO_PEER_TIMEOUT: u64 = 200;
```

Comment:
Since this value does not inherently have units, I would add a `_MS` suffix to the name for clarity.

Reply:
done